etc/openwrt/openthread-br/Makefile

```diff
@@ -43,16 +43,23 @@ CMAKE_SOURCE_DIR=$(LOCAL_SOURCE_DIR)
 CMAKE_OPTIONS+= \
 	-DBUILD_TESTING=OFF \
 	-DCMAKE_INSTALL_PREFIX=/usr \
-	-DOTBR_BORDER_AGENT=OFF \
-	-DOTBR_MDNS=OFF \
+	-DOTBR_OPENWRT=ON \
+	-DOTBR_BORDER_AGENT=ON \
+	-DOTBR_BORDER_ROUTING=ON \
+	-DOTBR_SRP_ADVERTISING_PROXY=ON \
+	-DOTBR_MDNS="avahi" \
+	-DOTBR_INFRA_IF_NAME=\"br-lan\" \
+	-DOT_POSIX_SETTINGS_PATH=\"/etc/openthread\" \
 	-DOT_READLINE=OFF \
-	-DOTBR_OPENWRT=ON
+	-DOT_FIREWALL=ON
+
+TARGET_CFLAGS += -DOPENTHREAD_POSIX_CONFIG_DAEMON_SOCKET_BASENAME=\\\"/var/run/openthread-%s\\\"
 
 define Package/openthread-br
 	SECTION:=base
 	CATEGORY:=Network
 	TITLE:=OpenThread Border Router
-	DEPENDS:=+libstdcpp +libjson-c +libubus +libblobmsg-json
+	DEPENDS:=+libstdcpp +libjson-c +libubus +libblobmsg-json +libavahi-client +iptables-mod-extra +ipset
```

Comment:
should we add avahi-daemon as the runtime dependency as well?

Reply:
Yes, we have to! I will do it